### DISCUSSION TITLE
Spotify connect issue

### DISCUSSION THREAD

**Discussion by hellomiakoda** [kraxarn/spotify-qt#278] [Q&A]
I know "Device" used to show other Spotify connect devices on my network. Now it only sees LibreSpot on this computer. The official Spotify app shows my Home Assistant Server and 2 PlayStations. I know this used to work, not sure when it stopped working. Any idea what I'm doing wrong?

**Comment by kraxarn** [comment]
I currently have 2 different speakers with Spotify Connect on my network, and they work perfectly fine with spotify-qt, so I dunno what's wrong in your case, sorry :/

**Reply by iamboozer** [reply]
I'm experiencing something similar as @hellomiakoda with my AVR while using spotify-qt. Everything seems to be fine with a single user, but this AVR isn't shown under "Device" to another Spotify user while using spotify-qt.

The official Spotify app discovers my AVR just fine between users, and after the other user has connected to it with the official app it's shown under "Device" by spotify-qt too.

I'm using spotify-qt with librespot, and librespot is configured to run as a user service.

I've also tried with my firewall and VPN disabled (my LAN is whitelisted in my VPN client settings), but this made no difference.
